types/tls.go

```diff
@@ -183,3 +166,26 @@ var (
 		TLS_RSA_WITH_AES_256_CBC_SHA:                  "AES256-SHA",
 	}
 )
+
+func (c *TLSCipherSuite) String() string {
+	return fmt.Sprintf("%s", *c)
+}
```

Comment:
Same as above

Reply:
alternatively `return string(*c)`

Reply:
Also if you never make a pointer value of this `TLSCipherSuite` you can just use the simpler non-pointer receiver:
```
func (c TLSCipherSuite) String() string { return string(c) }
```